### PR TITLE
Add Active Collab

### PR DESCRIPTION
**Pull request by nurtext**
Unofficial desktop application for [ActiveCollab](https://www.activecollab.com).

### CODE REVIEW

**Comment by sindresorhus** [comment]
In your readme, you should link to ActiveCollab. Coming to your repo I have no idea what it is.

Can you fix the merge conflict?

File: readme.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ Some good apps written with Electron.
 - [Fog](https://github.com/vitorgalvao/fog) - Unofficial Overcast podcast app.
 - [Wmail](https://github.com/Thomas101/wmail) - Unofficial Gmail & Google Inbox app.
 - [Boostnote](https://github.com/BoostIO/Boostnote) - Markdown note app for developers.
+- [Active Collab Desktop](https://github.com/nurtext/active-collab-desktop) - Unofficial Active Collab Desktop application.
```
Comment (sindresorhus) [inline readme.md]: `Desktop` is moot as its implied. `application` => `app` for consistency.

Comment (nurtext) [inline readme.md]: Changed.

**Comment by nurtext** [comment]
@sindresorhus Fixed conflict.

**Pull request closed without merging**